experimental/graphviz/handle.go

```diff
@@ -14,7 +14,7 @@ import (
 func handleTXT(r Reporter) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
 		w.Header().Set("Content-Type", "text/plain")
-		dot(w, r.Report().Process.RenderBy(mapFunc(req), classView(req)))
+		//dot(w, r.Report().Process.RenderBy(mapFunc(req), nil, classView(req)))
```

Comment:
Can you fix this properly? If you don't want to bother, that's fine — in which case I think it's better to just temporarily remove graphviz from the Makefile or whatever.